<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFeedback::Analyzer::MainWindow</class>
 <widget class="QMainWindow" name="UserFeedback::Analyzer::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Feedback Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTableView" name="dataView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionConnectToServer"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Products">
    <property name="title">
     <string>Pro&amp;ducts</string>
    </property>
    <addaction name="actionAddProduct"/>
    <addaction name="actionDeleteProduct"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Products"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="logDock">
   <property name="windowTitle">
    <string>&amp;Log Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="logWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="productsDock">
   <property name="windowTitle">
    <string>P&amp;roducts</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeView" name="productListView">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionConnectToServer">
   <property name="text">
    <string>&amp;Connect to server...</string>
   </property>
   <property name="toolTip">
    <string>Connect to analytics server.</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the user feedback analyzer.</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAddProduct">
   <property name="text">
    <string>&amp;Add Product...</string>
   </property>
   <property name="toolTip">
    <string>Add a new product.</string>
   </property>
  </action>
  <action name="actionDeleteProduct">
   <property name="text">
    <string>&amp;Delete Product</string>
   </property>
   <property name="toolTip">
    <string>Delete product and all associated data.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
